src/docdb/getDocumentClient.ts

```diff
@@ -3,27 +3,18 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { DocumentClient } from "documentdb";
-import * as DocDBLib from 'documentdb/lib';
-import * as vscode from "vscode";
+import { CosmosClient } from "@azure/cosmos";
 import { appendExtensionUserAgent } from "vscode-azureextensionui";
-import { ext } from "../extensionVariables";
 
-export function getDocumentClient(documentEndpoint: string, masterKey: string, isEmulator: boolean | undefined): DocumentClient {
-    const documentBase = DocDBLib.DocumentBase;
-    const connectionPolicy = new documentBase.ConnectionPolicy();
-
-    const vscodeStrictSSL: boolean | undefined = vscode.workspace.getConfiguration().get<boolean>(ext.settingsKeys.vsCode.proxyStrictSSL);
-    const strictSSL = !isEmulator && vscodeStrictSSL;
-    connectionPolicy.DisableSSLVerification = !strictSSL;
-    const client = new DocumentClient(documentEndpoint, { masterKey: masterKey }, connectionPolicy);
+export function getDocumentClient(endpoint: string, key: string): CosmosClient {
+    const client = new CosmosClient({ endpoint, key });
```

Comment:
> "DisableSSLVerification" is no longer available with CosmosClient. I am looking at ways to bypass ssl for the new CosmosClient to support Emulator usage. I thought of creating an issue for this and move along with the current PR, what do you think?

Yes I think that's fine - just make extra sure we fix that bug before releasing because it would represent a regression if we didn't